commands/article.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config.json');
+
+exports.command = async (message, args, channels, database) => {
+
+  let query = args.join(' ').toLowerCase();
+
+  if(!query){
+    await message.channel.send('Please provide a search query');
+    return ;
+  }
+  const axios = require('axios');
+
+  let response = await axios.get('https://aternos.zendesk.com/api/v2/help_center/articles/search.json?query='+query)
```

Comment:
urlencode query in Zendesk API URL
```suggestion
  let response = await axios.get('https://aternos.zendesk.com/api/v2/help_center/articles/search.json?query='+encodeURIComponent(query))
```